src/main/java/online/book/store/controller/BookController.java

```diff
@@ -5,11 +5,15 @@
 import online.book.store.dto.BookDto;
 import online.book.store.dto.CreateBookRequestDto;
 import online.book.store.service.BookService;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.DeleteMapping;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
 @RequiredArgsConstructor
```

Comment:
check the task description, it says that our endpoints should be not just the `/books` but `/api/books`